docs/pages/toolchain/katana/genesis.md

```diff
@@ -63,16 +63,16 @@ The genesis config file is a JSON file that contains the following fields:
 - `contracts` _Genesis contract deployments._
   - <CONTRACT_ADDRESS> _The address of the contract._
     - `class`  
-      _The class of the contract._
+      _The name of the class._
```

Comment:
this field can either be the class hash (0x-prefixed hex value) or name. Corresponds to the `classHash`, and `name` fields of the `classes` list respectively. As seen here 

https://github.com/dojoengine/dojo/blob/6ae661d4e272c0c0920858e255a5fd89aa40ad53/crates/katana/primitives/src/genesis/json.rs#L177-L182

this is the same for every `class` field in [fee token](https://github.com/dojoengine/book/blob/0b2d6d8912443bb7ac71e43c543ad7c0a606cd20/docs/pages/toolchain/katana/genesis.md?plain=1#L40), [account](https://github.com/dojoengine/book/blob/0b2d6d8912443bb7ac71e43c543ad7c0a606cd20/docs/pages/toolchain/katana/genesis.md?plain=1#L59), and [contract](https://github.com/dojoengine/book/blob/0b2d6d8912443bb7ac71e43c543ad7c0a606cd20/docs/pages/toolchain/katana/genesis.md?plain=1#L65) config entries